cmd/swarm/config_test.go

```diff
@@ -296,6 +286,14 @@ func TestConfigCmdLineOverrides(t *testing.T) {
 		t.Fatalf("Expected Cors flag to be set to %s, got %s", "*", info.Cors)
 	}
 
+	if info.SwapPaymentThreshold != swap.DefaultPaymentThreshold {
```

Comment:
You are testing that if the thresholds are not set that the default will be taken here right? While this sounds like a good idea, I wonder if we better (or also) need one where it tests that if we set a specific value that that is taken.

Reply:
Actually, this test was supposed to test that command-line parameters overwrite the default values, but I see that adding a default-value as a command-line parameter obviously doesn't do anything meaningless. 

Changed it to reflect the actual meaning of the test.